src/main/java/org/dreambot/quests/cooksassistant/GatherCooksAssistantItemsLeaf.java

```diff
@@ -75,7 +73,7 @@ public int onLoop() {
                 if(!WHEAT_FIELD_AREA.contains(Players.getLocal()))
                 {
                     if(Walking.shouldWalk(6)) {
-                        Walking.walk(WHEAT_FIELD_AREA.getRandomTile());
+                        Interaction.delayWalk(WHEAT_FIELD_AREA.getRandomTile());
```

Comment:
I think we should not clutter one class with many actions. You can separate out the gathering of items such as RetrieveEggLeaf, RetrieveMilkLeaf etc for all the items. Makes it more readable and easy to update if there are quest changes. (Probably not likely for this particular quest).